lib/aruba/processes/basic_process.rb

```diff
@@ -115,16 +115,13 @@ def before_run; end
       def after_run; end
 
       def inspect
-        out = stdout(:wait_for_io => 0) + stderr(:wait_for_io => 0)
+        out = truncate("#{stdout(:wait_for_io => 0).inspect}", 35)
+        err = truncate("#{stderr(:wait_for_io => 0).inspect}", 35)
```

Comment:
It was 76 for both input + output. So I just picked 35+35=70 + six characters left for another variable in the inspect string.

> Strictly speaking this is an API change. 

It isn't - no one is relying on the output of `#inspect` to rely on any schema.

>Normally terminals should be 80 characters wide.

Yes, but the length here is completely arbitrary - it was meant to just reduce 3-pages worth of instance variable dumps during an exception (BasicProcess class just has so much inside).

See here: 1d6a0ef4b4ae98dce14e6ec17485ce11a588e10d

The previous inspect string format is too long by itself anyway: `'#<%s:%s commandline="%s": output="%s">'` -> 38 characters, meaning only 42 would be left for the output string.

38 + 76 = 114

after my changes:

46 + 35 + 35 = 116

Again, it's just to reduce 3 pages of dumped internal variables into something a little more "user friendly".

Not to mention now you at least have a chance to see part of the actual error! (Previously, by appending the errors on stdout, you wouldn't see even a single character from stderr if stdout produced more than 76 characters...).

It's just semantically wrong to concat stdout and stderr, no matter what the use case is. (They're separate streams for strong reasons).